db/seeds.rb

```diff
@@ -6,32 +6,68 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Category.create([{ title: 'Apparel' }, { title: 'Snowboards' }])
+Category.create([{ title: 'Apparel' }, { title: 'Snowboards' }, { title: 'Boots' }, { title: 'Bindings' }])
 
-Category.first
+apparel = Category.find_by(title: "Apparel")
   .items
   .create([{ name: 'Hoody',
              description: 'Keeps you slightly warm',
              price: 80,
+             brand: "Nomis",
              image: File.open('app/assets/images/hoody.jpg')
                  },
 
            { name: 'Sweet Jacket',
              description: 'Keeps you warm',
              price: 240,
+             brand: "NEFF",
              image: File.open('app/assets/images/jacket.jpg')
             }])
 
-Category.last
+snowboards = Category.find_by(title: "Snowboards")
   .items
   .create([{ name: 'gwar possum',
              description: 'a snowboard for gwar concerts',
              price: 15,
+             brand: "Jibberish",
              image: File.open('app/assets/images/gwar_possum.jpg')
                  },
 
            { name: 'gnar possum',
              description: 'a snowboard for shredding gnar pow',
              price: 1000,
+             brand: "Never Summer",
              image: File.open('app/assets/images/gnar_possum.jpg')
            }])
+
+boots = Category.find_by(title: "Boots")
+.items
+.create([{ name: 'cool boots',
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.